crochet-stitcher/src/types.ts

```diff
@@ -26,6 +26,18 @@ export enum StitchType {
     Treble = 'Treble',
 }
 
+export enum Colour {
+    White = 'white',
```

Comment:
I wonder if the keys could be changed to match the values, like `White = 'White'`?